src/coreclr/jit/compiler.cpp

```diff
@@ -27,9 +27,6 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 extern ICorJitHost* g_jitHost;
 
 #ifdef TARGET_WASM
-#undef min
```

Comment:
This is interesting, were they causing a problem?  They were part of the problem with the linux build if I remember correctly.

Reply:
Not really, but `llvm.h` already does this for LLVM headers to compile and there is no need to spread the workaround elsewhere.

(It would be ideal to redefine them after the `#include`s in `llvm.h` itself, so we don't need to duplicate that, but that's a task for another day)